internal/charmstore/store.go

```diff
@@ -215,7 +222,10 @@ func (p *Pool) requestStoreNB(always bool) (*Store, error) {
 		bp.Store = macStore
 		bsvc, err := bakery.NewService(bp)
 		if err != nil {
-			return nil, errgo.Notef(err, "cannot make bakery service")
+			// This should never happen because the only reason bakery.NewService
```

Comment:
This in not guaranteed for all future versions, but it is reasonable to deal with that at a later time.